components/google_sheets/google_sheets.app.mjs

```diff
@@ -86,6 +90,23 @@ export default {
   },
   methods: {
     ...googleDrive.methods,
+    sanitizedArray(value) {
+      if (isArray(value)) {
+        return value.map((item) => get(item, "value", item));
+      }
+
+      // If is string, try to convert it in an array
+      if (isString(value)) {
+        // Return an empty array if string is empty
+        if (isEmpty(value)) {
+          return [];
+        }
+
+        return value.replace(/["'[\]\s]+/g, "").split(",");
```

Comment:
In the previous one, it works only if you add a perfect array-like string. I think we can remove the space from the regex. I will check if I can do something related to the `[]`

Reply:
If we remove whitespace from the regex, `["foo", "bar"]` would be invalid. It would need to be `["foo","bar"]`, with no space. Do you think whitespace should be allowed between array elements and commas/brackets?